ui/src/containers/SampleListViewContainer.jsx

```diff
@@ -77,6 +82,20 @@ export default function SampleListViewContainer() {
     handleSetViewMode(localStorageViewMode || viewMode.mode);
   }, [handleSetViewMode, viewMode.mode]);
 
+  const getSamplesFromSC = useCallback(() => {
+    dispatch(getSamplesList());
+  }, [dispatch]);
+
+  useEffect(() => {
```

Comment:
I could also do like this. 
```suggestion
    useEffect(() => {
    if (!showClearSamplesButton) {
      getSamplesFromSC();
    }
  // eslint-disable-next-line react-hooks/exhaustive-deps
  }, []);
```